projects/noloan/app/src/main/java/hasadna/noloan/firestore/FirestoreClient.java

```diff
@@ -54,18 +55,36 @@ public TaskCompletionSource StartListeningSpam() {
             } catch (InvalidProtocolBufferException e1) {
               e1.printStackTrace();
             }
-            switch (dc.getType()) {
-              case ADDED:
-                sp.add(sms);
-                break;
-              case MODIFIED:
-                sp.modified(sms);
-                break;
-              case REMOVED:
-                sp.remove(sms);
-                break;
+            // Spam list
+            if (path.equals(SPAM_COLLECTION_PATH)) {
+              switch (dc.getType()) {
```

Comment:
I know the code was like that before, but I think `dc->documentChange` would be better.

Reply:
Yip, I think so too.
Done.